src/groupeddataframe/grouping.jl

```diff
@@ -190,42 +231,100 @@ end
 
 """
     combine(gd::GroupedDataFrame)
-    combine(f::Function, gd::GroupedDataFrame)
+    combine(gd::GroupedDataFrame, cols => f...)
+    combine(gd::GroupedDataFrame; (colname = cols => f)...)
+    combine(gd::GroupedDataFrame, f)
+    combine(f, gd::GroupedDataFrame)
```

Comment:
In line 237 `f` can be also a tuple or a named tuple. Right?

Reply:
So the question left if `f` can be a 'Pair'?

Reply:
Ah, yes. I've fixed that.